HomeWorks/src/main/java/com/epam/spring/homework2/beans/validator/FieldsValidator.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.spring.homework2.beans.validator;
```

Comment:
Optional extra item from me - try implementing validation via visitor design pattern.

Reply:
I tried to implement this pattern and I came across an info. Is it real problem?
![image](https://user-images.githubusercontent.com/44750080/151698485-df262121-74b2-44fc-984c-9bc5592f4eee.png)